tests/fixtures/v430comlink.html

```diff
@@ -0,0 +1,46 @@
+<script type="module">
+  // comlink main
+  import * as Comlink from "/base/dist/esm/comlink.mjs";
+
+  window.addEventListener("message", function l(ev) {
+    if (
+      ev.data &&
+      typeof ev.data === "object" &&
+      ev.data.constructor.name === "MessagePort"
+    ) {
+      window.removeEventListener("message", l);
+      register(ev.data);
+    }
+  });
+
+  function register(parentEndpoint) {
+    class Test {
+      async acceptProxy(aProxy) {
+        return (await aProxy()) + 1;
+      }
+
+      callme(port) {
+        Comlink.wrap(port).maybe();
+      }
```

Comment:
Note that this code does not care whether instances of `Test` are exposed to an endpoint that need to talk 4.3.0 format or the latest format.

The trick is that when `port` is being received here, we've "marked" it so that if any code path tries to `wrap` it, the code will automatically work.

If that trick does not work because the port is nested inside an object, you can always resort to passing `Comlink.wrap(port, undefined, true)` manually which would force the proxy created to use the `4.3.0` format.